airbyte-integrations/bases/source-acceptance-test/source_acceptance_test/utils/json_schema_helper.py

```diff
@@ -24,11 +24,45 @@
 
 
 from functools import reduce
-from typing import Any, List, Set
+from typing import Any, List, Mapping, Set
 
 import pendulum
 
 
+class Field:
```

Comment:
I like this class!

maybe a better name for this class would be `CatalogField`? Just trying to avoid conflict with `Field` which is already used by pydantic